content/cpp/concepts/math-functions/terms/nearbyint/nearbyint.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'nearbyint()'
+Description: 'Returns an integer that is closest to a given argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`nearbyint()`** function returns an integer that is closest to a given floating point argument.
+
+## Syntax
+
+```pseudo
+nearbyint(n)
+```
+The function accepts an `n` parameter that represents a positive or negative floating point value. If the tenths decimal value of `n` is 5 or greater, then the function returns an integer value greater than `n`.
```

Comment:
"The function accepts a parameter `n`..."
"If the decimal value of `n` is 0.5 or greater..."

Reply:
I have resolved your comments. Thanks for reviewing.